api/src/main/java/org/openmrs/Patient.java

```diff
@@ -234,7 +198,106 @@ public void setPersonId(Integer personId) {
 	public void setIdentifiers(Set<PatientIdentifier> identifiers) {
 		this.identifiers = identifiers;
 	}
-	
+
+	@ManyToOne
+	@JoinColumn(name = "creator", updatable = false)
+	@Access(AccessType.PROPERTY)
+	@Override
+	public User getCreator() {
+		return super.getCreator();
+	}
+
+	@Override
+	public void setCreator(User creator) {
```

Comment:
Do you really need to override the setters if all you are doing is call the method from the superclass?

Reply:
@wluyima I don't think so.  Originally I just overrode the getters and that fixed the main problem I was seeing.  I tried overriding the setters to see if it would fix the other issue I was seeing, but it didn't seem to make a difference, so, you are right, we should revert that part.